src/payment/components/ProductLineItem.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from '@edx/frontend-i18n';
+
+
+class ProductLineItem extends React.PureComponent {
+  renderSeatType(seatType) {
+    const selectedSeatType = (() => {
```

Comment:
You could refactor this to "unwrap" this anonymous function. renderSeatType isn't doing much more. You could put the <p></p> stuff back in the main render function